Adyen/Components/Base/Component.swift

```diff
@@ -29,6 +33,17 @@ public extension Component {
         }
     }
     
+    /// :nodoc:
```

Comment:
We can actually add here some user-friendly description. This is something that component implementers could possibly use..

Reply:
there is documentation in the `Component` protocol for the `clientKey` var, right? we have kind of a convention that if a class or struct inherits from/implements a class/protocol, the overriding declaration/implementation should show :nodoc:, as the parent class/protocol has the documentation, so this falls under this rule/convention I guess.